week-4/Homework/mandatory/4-groceries.js

```diff
@@ -6,7 +6,9 @@
 let groceriesToBuy = [];
 
 let groceryList = {
-  item1: "",
-  item2: "",
-  item3: ""
+  item1: "Potatoes",
+  item2: "Orange Juice",
+  item3: "Rice"
 };
+
+console.log();
```

Comment:
Part one is done here, but can you finish off part 2 of this, to loop through and add the items to the array?

Thanks!